cabal-install-solver/cabal-install-solver.cabal

```diff
@@ -39,18 +39,18 @@ library
   default-language: Haskell2010
   hs-source-dirs:   src
   hs-source-dirs:   src-assertion
-  ghc-options:      -Wall -fwarn-tabs -fwarn-incomplete-uni-patterns
-
-  if impl(ghc >=8.0)
-    ghc-options: -Wcompat -Wnoncanonical-monad-instances
-
-    if impl(ghc <8.8)
-      ghc-options: -Wnoncanonical-monadfail-instances
+  ghc-options:
+    -Wall -Wcompat -Wnoncanonical-monad-instances
+    -fwarn-tabs -fwarn-incomplete-uni-patterns
 
+  if impl(ghc <8.8)
+    ghc-options: -Wnoncanonical-monadfail-instances
     if impl(ghc >=8.10)
       ghc-options: -Wunused-packages
```

Comment:
The inner condition is always false. I think it's one level too deep

Reply:
Fixed